types/src/front/lib/connectors_api.ts

```diff
@@ -18,6 +18,9 @@ const CONNECTORS_ERROR_TYPES = [
   "oauth_token_revoked",
   "third_party_internal_error",
   "webcrawling_error",
+  "webcrawling_error_empty_content",
```

Comment:
question of whether to refactor this so that syncFailed can have specific errors without making this list too long => for me it's ok to leave it out of scope of the PR, can be done later